README.md

```diff
@@ -25,7 +25,7 @@ int main(int argc, char** argv)
     Vec x = linspace(0.0, 5.0, 100);
 
     // Create a Plot object
-    Plot plot;
```

Comment:
I think the vast majority of users will require just 2D plots so that `Plot` is more convenient than `Plot2D`. Let those requiring 3D plots write two extra chars (`Plot3D`)!

Reply:
It is only two characters and clearly states the intent to have a 2D or 3D plot. Also PlotBase sucks as a name (when you're a user), so I tend to disagree.
BUT.
If we get rid of the two plot classes we might kill two or even more birds with one stone...